doc/sphinx/conf.py

```diff
@@ -187,10 +187,6 @@ def setup(app):
 nitpicky = True
 
 nitpick_ignore = [ ('token', token) for token in [
-    'induction_clause',
-    'where',
-    'bindings_with_parameters',
-    'destruction_arg'
 ]]
```

Comment:
Remove this entirely. We do not want to ease the re-introduction of exceptions.

Reply:
Sorry, `nitpicky = True` should stay. What should be removed is the entire `nitpick_ignore` definition.